x-pack/plugins/transform/public/app/sections/transform_management/components/edit_transform_flyout/edit_transform_flyout_form.tsx

```diff
@@ -365,6 +365,22 @@ export const EditTransformFlyoutForm: FC<EditTransformFlyoutFormProps> = ({
               }
             )}
           />
+          <EditTransformFlyoutFormTextInput
```

Comment:
I think you should also the value if set in the details section along with the other advanced settings.

![image](https://user-images.githubusercontent.com/7405507/176707687-619672bf-1150-45ae-a2e8-9111a8f507d5.png)

Reply:
Updated here https://github.com/elastic/kibana/pull/135486/commits/82590d2bd0a74410bab1a28ccdc787c48ed168a0